testing/src/main/java/io/stargate/it/http/RestApiMetricsTest.java

```diff
@@ -50,9 +49,13 @@
   // TODO
   @SuppressWarnings("unused") // referenced in @StargateSpec
   public static void buildParameters(StargateParameters.Builder builder) {
-    builder.putSystemProperties(
-        TestingServicesActivator.HTTP_TAG_PROVIDER_PROPERTY,
-        TestingServicesActivator.TAG_ME_HTTP_TAG_PROVIDER);
+    // 13-Jan-2022, tatu: In StargateV1 HTTP Tag Provider is registered using OSGi.
+    //    SGv2 does not have equivalent mechanism implemented yet, so tag functionality
+    //    NOT verified currently.
+    /*builder.putSystemProperties(
```

Comment:
This is a problem, we need to find a solution for this.. Not sure if we should postpone this, but I don't feel comfortable to disable tests and let this go in.. We will forget about it later on.. 

However, I have no idea how to do this now.. Maybe we should sync here..

Reply:
My primary concern right now is to get the integration test to completely pass, since this helps catch many other regressions. It'd be great to keep tags working if possible, but I think that since we also need to tackle change of frameworks, perhaps metrics part would be part of this effort -- the way service is boostrapped/initialized will change a bit anyway so making things work with DropWizard here may be work that does not help making things work with the newer framework.

Still, yes, I think we should sync.

Reply:
And just to make it clear: since tags are added in (and maybe used) Prod, this functionality would need to be fixed/added back before SGv2/REST would go in Prod. So disabling of testing, if any, should be temporary.
And not "just leave it out".